gulp/tasks/upload-preview.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    'use strict';
+
+    var gulp = require('gulp'),
+        gitUtils = require('../util/gitUtils'),
+        childProcess = require('child_process'),
+        previewDomain = 'ux-test.edx.org',
+        previewSiteDir = '_preview_site';
+
+    gulp.task('upload-preview', function() {
+        var branch = gitUtils.currentBranch();
+        childProcess.execSync(
+            'aws s3 sync ' + previewSiteDir + ' s3://' + previewDomain + '/' + branch
```

Comment:
Similar point to above - perhaps we could refactor this command into a config file or command line argument such that folks using non-s3 for hosting can upload previews as well.